views/signup.handlebars

```diff
@@ -103,4 +103,4 @@
     <script src="/assets/js/client.js"></script>
 </body>
 
-</html>
\ No newline at end of file
+</html>
```

Comment:
It looks like you're trying to use a different template engine here (`<%...`) is not handlebars style.